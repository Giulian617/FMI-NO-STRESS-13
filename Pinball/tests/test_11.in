8 149
1 3 9
6 1 3
2 6 6
8 2 8
7 8 8
4 7 4
5 4 5
