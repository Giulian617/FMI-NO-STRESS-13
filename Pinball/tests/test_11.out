250000002
750000006
